dogen/plugins/cct.py

```diff
@@ -91,6 +91,14 @@ def prepare(self, cfg):
 
         os.makedirs(target_modules_dir)
 
+        local_modules_dir = os.path.join(os.path.dirname(self.descriptor), 'cct')
+        self.log.debug('Found existing modules in directory: %s' % local_modules_dir)
+        if os.path.exists(local_modules_dir):
+            for module in os.listdir(local_modules_dir):
+                module_path = os.path.join(local_modules_dir, module)
+                self.log.info('Using cached module "%s" from path:"%s"' % (module, module_path))
```

Comment:
And in general - we use singel quote to quite things in strings. You did it other way.